client/src/List.svelte

```diff
@@ -194,7 +194,7 @@
 			}
 			if (changed) {
 				console.log("local adjustments changed");
-				localAdjustments.set(la);
+				localStorage.setItem("localAdjustments", JSON.stringify(la));
```

Comment:
There is a subscription in the store for local database events that we can make use of here. My understanding that the framework does not like our taking a "shortcut" setting the state directly with `localAdjustments.set`

Reply:
On the other hand, this [subscription](https://github.com/jonhoo/wewerewondering/blob/f010bc83313c532c3db9a6d4e7b0f192a776d5d1/client/src/store.js#L171-L178) can trigger endless [recursion](https://github.com/jonhoo/wewerewondering/blob/f010bc83313c532c3db9a6d4e7b0f192a776d5d1/client/src/store.js#L32-L38).

Reply:
Hmm, but that's what the `if (changed)` was intended for — that we don't keep going changing this even if nothing changed in the underlying data. I'm surprised the change as it currently is in MR doesn't _always_ trigger endless recursion? Did you try it with two browser windows where one is in host view and the other in client view?

Reply:
It's not that we are removing `if (changed)`, we are removing `localAdjustments.set(la);`  entirely, the framework does not allow this `set` mutation. And I've double checked with host and guest windows open - the app is working as expected, seems to me, and the `localAdjustments` and well as other entities are being updated on the users' disks.

Reply:
Yes direct assignments are changing the state, and - via subscriptions - end up in the local storage. 

As for testing, I just viewed how the data gets eventually consistent in the state, but - indeed - this happens soon (within 10 seconds for guests) and always, since the data is not cached. Hmm I think this can be arranged though..